homeassistant/components/aemet/sensor.py

```diff
@@ -51,172 +83,262 @@
+        key=f"forecast-daily-{ATTR_API_FORECAST_TIME}",
+        keys=[AOD_TOWN, AOD_FORECAST_DAILY, AOD_FORECAST_CURRENT, AOD_TIMESTAMP],
+        name="Daily Forecast Time",
         device_class=SensorDeviceClass.TIMESTAMP,
     ),
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_WIND_BEARING,
-        name="Wind bearing",
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_TIME}",
+        keys=[AOD_TOWN, AOD_FORECAST_HOURLY, AOD_FORECAST_CURRENT, AOD_TIMESTAMP],
+        name="Hourly Forecast Time",
+        device_class=SensorDeviceClass.TIMESTAMP,
+    ),
+    AemetSensorEntityDescription(
+        key=f"forecast-daily-{ATTR_API_FORECAST_WIND_BEARING}",
+        keys=[AOD_TOWN, AOD_FORECAST_DAILY, AOD_FORECAST_CURRENT, AOD_WIND_DIRECTION],
+        name="Daily Forecast Wind bearing",
         native_unit_of_measurement=DEGREE,
     ),
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_WIND_MAX_SPEED,
-        name="Wind max speed",
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_WIND_BEARING}",
+        keys=[AOD_TOWN, AOD_FORECAST_HOURLY, AOD_FORECAST_CURRENT, AOD_WIND_DIRECTION],
+        name="Hourly Forecast Wind bearing",
+        native_unit_of_measurement=DEGREE,
+    ),
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_WIND_MAX_SPEED}",
+        keys=[AOD_TOWN, AOD_FORECAST_HOURLY, AOD_FORECAST_CURRENT, AOD_WIND_SPEED_MAX],
+        name="Hourly Forecast Wind max speed",
         native_unit_of_measurement=UnitOfSpeed.KILOMETERS_PER_HOUR,
         device_class=SensorDeviceClass.WIND_SPEED,
     ),
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_WIND_SPEED,
-        name="Wind speed",
+    AemetSensorEntityDescription(
+        key=f"forecast-daily-{ATTR_API_FORECAST_WIND_SPEED}",
+        keys=[AOD_TOWN, AOD_FORECAST_DAILY, AOD_FORECAST_CURRENT, AOD_WIND_SPEED],
+        name="Daily Forecast Wind speed",
+        native_unit_of_measurement=UnitOfSpeed.KILOMETERS_PER_HOUR,
+        device_class=SensorDeviceClass.WIND_SPEED,
+    ),
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_WIND_SPEED}",
+        keys=[AOD_TOWN, AOD_FORECAST_HOURLY, AOD_FORECAST_CURRENT, AOD_WIND_SPEED],
+        name="Hourly Forecast Wind speed",
         native_unit_of_measurement=UnitOfSpeed.KILOMETERS_PER_HOUR,
         device_class=SensorDeviceClass.WIND_SPEED,
     ),
 )
-WEATHER_SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
-    SensorEntityDescription(
+
+
+WEATHER_SENSORS: Final[tuple[AemetSensorEntityDescription, ...]] = (
+    AemetSensorEntityDescription(
         key=ATTR_API_CONDITION,
+        keys=[AOD_WEATHER, AOD_CONDITION],
         name="Condition",
```

Comment:
```suggestion
        name="Condition",
        value_fn: lambda value: CONDITIONS_MAP.get(value)
```

Reply:
Done in f064f9e010966a5c9832823cc324e04c7e348d2a